biz.aQute.bndlib.tests/test/test/MacroTest.java

```diff
@@ -1376,6 +1377,85 @@ public void testSystemAllowFail() throws Exception {
 		}
 	}
 
+	@Test
+	@DisabledOnOs(WINDOWS)
+	public void testSystemInPath() throws Exception {
+		try (Processor p = new Processor()) {
+			Macro macro = new Macro(p);
+			assertEquals("/tmp", macro.process("${system-in-path;/tmp;pwd}"));
```

Comment:
Instead of a hard coded path, you should create a folder in the `@InjectTemporaryDirectory` temp folder. Then these test can run on all OS.